### ISSUE TITLE
Upload Data Page

### ISSUE THREAD

**Issue by hamzahalilovic** [prifina/app-desktop#28]
**Upload Data Page**

The instructions to upload data.

A page appears with instructions on uploading data.

**Includes:** Instructions for Uploading Data

**Relevant user story:** [https://docs.google.com/document/d/148Oh7AnmXeJ-IBpVWqSGH7jgRo9wNLOjXBnUXeJ9tvg/edit?ts=5f87335e#heading=h.o6do42inilxx](url)
**Visual component design:** [https://www.figma.com/file/fS9gcgUb0KShgtZ3XIYQqY/Production-Version-Software-Development?node-id=964%3A699](url)

**Specifications:**

**Comment by mmlampinen** [comment]
To clarify this component, **it does not include**: header bar, left menu, right menu cc @hamzahalilovic

**Comment by mmlampinen** [comment]
> To clarify this component, **it does not include**: header bar, left menu, right menu cc @hamzahalilovic

Actually, I just noticed there's also this: https://github.com/prifina/app-desktop/issues/29 

So maybe this #28 should be 0.44? @hamzahalilovic

**Comment by hamzahalilovic** [comment]
@mmlampinen Yep this was made like a page container in which "upload data element" is inserted. We can close this issue maybe?

**Comment by mmlampinen** [comment]
I think we need both, the first version (0.24) and the later version (0.44). Let's just move to 0.44 for now

**Comment by hamzahalilovic** [comment]
Sounds good
